src/inputType/PanInput.ts

```diff
@@ -188,7 +191,10 @@ export class PanInput implements IInputType {
 	*/
 	destroy() {
 		this.disconnect();
-		if (this.hammer && this.hammer.recognizers.length === 1) {
```

Comment:
Do you need to check the length of the recognizers when call 'destory' method?

before ```this.hammer = null;```, must call 'destory' method

Reply:
That condition is checking whether the other input type is not using same hammer instance like PinchInput.